src/state/actions/lang.js

```diff
@@ -0,0 +1,7 @@
+import { createAction } from 'redux-act';
+
```

Comment:
We should call this state `preferences` or `userPreferences` something like that so in the future we can store all the user preferences stuff here.